units/src/amount/unsigned.rs

```diff
@@ -59,37 +64,68 @@ impl Amount {
     /// Exactly one satoshi.
     pub const ONE_SAT: Self = Amount(1);
     /// Exactly one bitcoin.
-    pub const ONE_BTC: Self = Self::from_int_btc_const(1);
-    /// The maximum value allowed as an amount. Useful for sanity checking.
-    pub const MAX_MONEY: Self = Self::from_int_btc_const(21_000_000);
+    pub const ONE_BTC: Self = Self::from_sat_unchecked(100_000_000);
+    /// The maximum value allowed as an amount.
+    pub const MAX_MONEY: Self = Self::from_sat_unchecked(21_000_000 * 100_000_000);
     /// The minimum value of an amount.
     pub const MIN: Self = Amount::ZERO;
     /// The maximum value of an amount.
     pub const MAX: Self = Amount::MAX_MONEY;
     /// The number of bytes that an amount contributes to the size of a transaction.
     pub const SIZE: usize = 8; // Serialized length of a u64.
 
-    /// Constructs a new [`Amount`] with satoshi precision and the given number of satoshis.
+    /// Constructs a new [`Amount`] from the given number of satoshis.
+    ///
+    /// # Errors
+    ///
+    /// If `satoshi` is outside of valid range (greater than `self::MAX_MONEY`).
```

Comment:
Can this be change to  use greater than 21 million bitcoin instead of saying MAX_MONEY.